performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/RuntimeContentProvider.java

```diff
@@ -87,8 +87,11 @@ private Config createConfig(@NonNull final Context context,
 
     boolean appDebuggable = Util.isAppDebuggable(context);
     double enablePercent = lastConfig.getEnablePercent();
-    double randomNumber = new Random(System.currentTimeMillis()).nextDouble() * 100.0;
-    boolean enableTracking = (randomNumber <= enablePercent);
+    boolean enableTracking = false;
+    if (enablePercent > 0.0) {
```

Comment:
why do you need this? based on the `Random#nextDouble` docs:

> The general contract of nextDouble is that one double value, chosen (approximately) uniformly from the range 0.0d (**inclusive**) to 1.0d (**exclusive**)

so if change the previous condition to 
```
boolean enableTracking = (randomNumber < enablePercent);
```
you get the behavior:
* `enablePercent == 0` → `enableTracking == false` because `randomNumber` is never negative
* `enablePercent == 100` → `enableTracking == true` because `randomNumber` is never 100 (because `Random#nextDouble` is never `1.0d`)

that change would be more concise and correct (i think).